resource/constants.rs

```diff
@@ -127,7 +127,7 @@ pub mod snapshot {
 }
 
 pub mod storage {
-    pub const TIMELINE_WINDOW_SIZE: usize = 100;
+    pub const TIMELINE_WINDOW_SIZE: usize = 16;
```

Comment:
Higher contention on the lock might be bad for write performance. Can revert, but changed it to ask the question.